bin/check-asg-instances-created.rb

```diff
@@ -34,18 +34,18 @@
 require 'aws-sdk'
 require 'sensu-plugins-aws'
 
-class CheckAsgInstanceCreated < Sensu::Plugin::Check::CLI
+class CheckAsgInstancesCreated < Sensu::Plugin::Check::CLI
   include Common
   option :aws_region,
          short:       '-r AWS_REGION',
          long:        '--aws-region REGION',
          description: 'AWS Region (defaults to us-east-1).',
          default:     ENV['AWS_REGION']
 
-  option :asg_group_name,
-         short:       '-g G',
-         long:        '--asg_group_name AutoScalingGroupName',
-         description: 'AutoScalingGroupName to check'
+  option :asgname,
```

Comment:
reference:
https://github.com/phoppe93/sensu-plugins-aws/blob/master/bin/check-elb-health-fog.rb

better to use the option -n like in the reference.

@sstarcher